cms/blocks.py

```diff
@@ -16,3 +16,12 @@ class LearningTechniqueBlock(blocks.StructBlock):
 
     class Meta:
         icon = "plus"
+
+
+class TermsOfServicesBlock(blocks.StructBlock):
```

Comment:
Can we perhaps move to a more generic approach? Like a `ResourceBlock` or something, comprised of a `heading` and `body` or `content`? We would be able to reuse it in other places/pages.